bundles/org.openhab.core.model.yaml/src/main/java/org/openhab/core/model/yaml/internal/YamlModelRepositoryImpl.java

```diff
@@ -116,7 +117,8 @@ public void deactivate() {
     public synchronized void processWatchEvent(Kind kind, Path path) {
         Path fullPath = watchPath.resolve(path);
         String pathString = path.toString();
-        if (Files.isDirectory(fullPath) || fullPath.toFile().isHidden() || !pathString.endsWith(".yaml")) {
+        if (!Files.isReadable(fullPath) || Files.isDirectory(fullPath) || path.startsWith("automation")
```

Comment:
@lolodomo, @J-N-K - even though it's not a root cause, do you think it would make sense for performance reasons to explicitly ignore ".git" directories as well? See https://community.openhab.org/t/openhab-4-2-milestone-discussion/154316/185

I realize this could quickly turn into a rather large blacklist, though.

Reply:
No. That doesn't make sense. Another guy in thread thread had a directory `_infdb`, why should we add `.git` but not `_infdb`? `automation` is different because it is readable, but may contain incompatible `.yaml` files.